recipes/vmtouch/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "vmtouch" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hoytech/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d57b7b3ae1146c4516429ab7d6db6f2122401db814ddd9cdaad10980e9c8428c
+
+build:
+  skip: true  # [win]
+  number: 0
+  script:
+    - make
+    # "make install" is broken on Mac OS in version 1.3.1, but should be fixed
+    # in the next release: https://github.com/hoytech/vmtouch/pull/71
+    # Here we'll just include the fix manually.
+    - mkdir -p $PREFIX/bin $PREFIX/share/man/man8
```

Comment:
There is a discussion about conda provided man pages [here](https://github.com/conda/conda/issues/845). I think you're doing the "right" thing already.